src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
-public class MainApp {
+public class MainApp{
+    public static void main(String[] args) {
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+        Machine[] machina = new Machine[] {truck, bulldozer, excavator};
```

Comment:
I'd recommend creating truck, bulldozer, and excavator instances directly within array initializer in order to avoid creating redundant variables. Also, let's make your array name more clear